menas/src/main/scala/za/co/absa/enceladus/menas/controllers/SchemaController.scala

```diff
@@ -115,18 +121,18 @@ class SchemaController @Autowired() (
 
         val sparkStruct = StructType(sparkMenasConvertor.convertMenasToSparkFields(schema.fields))
         if (pretty) sparkStruct.prettyJson else sparkStruct.json
-      case None         =>
+      case None =>
         throw notFound()
     }
   }
 
   /**
-    * Parses an StructType JSON file contents and converts it to Spark [[StructType]].
-    *
-    * @param structTypeJson A StructType JSON string.
-    * @return The parsed schema as an instance of [[StructType]].
-    */
-  private def parseStructType(structTypeJson: String): StructType = {
+   * Parses an StructType JSON file contents and converts it to Spark [[StructType]].
+   *
```

Comment:
Avoid changing styles of comments and others. Otherwise, we can find ourselves in PRs full of mainly style changes.

Reply:
sorry, reverted. (I have autoformatted the file in IDEA thinking that this would be ok for everyone.)